isvctl/configs/suites/README.md

```
@@ -123,7 +123,9 @@ For the domain / script-count / AWS-reference overview see the
 | `query_health_aggregation` | test | `providers/nico/scripts/health/query_health_aggregation.py` | `aggregation_level`, `groups[].{total,healthy,unhealthy,status,unhealthy_hosts}` |
 | `query_ib_tenant_isolation` | test | `providers/nico/scripts/infiniband/query_ib_tenant_isolation.py` | `partitions_checked`, `partitions[].{name,partition_key,tenant_id,status}` |
 | `query_ib_keys` | test | `providers/nico/scripts/infiniband/query_ib_keys.py` | `partitions_with_pkey`, `keys.<name>.{configured,source,detail}` |
-| `query_sanitization` | test | `providers/nico/scripts/sanitization/query_sanitization.py` | `machines_checked`, `machines[].{available,in_use,has_gpu,served_tenant,sanitized,stale_tenant_binding,vendor,product_name,bios_version,transitions}` |
+| `query_sanitization` | test | `providers/nico/scripts/sanitization/query_sanitization.py` | `machines_checked`, `machines[].{available,in_use,has_gpu,served_tenant,sanitized,breakfix_skip_observed,tenancy_preserved,stale_tenant_binding,vendor,product_name,bios_version,transitions}` |
+| `query_stable_ips` | test | `providers/nico/scripts/storage/query_stable_ips.py` | `hosts_checked`, `hosts[].{host_id,hw_sku_device_type,primary_ip_addresses}` |
+| `query_oob_health` | test | `providers/nico/scripts/health/query_oob_health.py` | `hosts_checked`, `hosts[].{oob_health_present,bmc_probe_ids,failure_categories.<device|network|memory|drive>.{observable,probe_ids}}` |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 4; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)